src/shadowbox/README.md

```diff
@@ -1,203 +1,169 @@
-```
+   - **Get an access key (e.g. ID 1):** `curl --insecure $API_URL/access-keys/1`
 
-Get an access key (e.g. get access key 1)
+   - **Rename an access key:** `curl --insecure -X PUT -F 'name=albion' $API_URL/access-keys/2/name`
 
-```
-curl --insecure $API_URL/access-keys/1
-```
+   - **And more...**
 
-Rename an access key
-(e.g. rename access key 2 to 'albion')
+1. **Further Options:**
 
-```
-curl --insecure -X PUT -F 'name=albion' $API_URL/access-keys/2/name
-```
+   Consult the [OpenAPI spec](./server/api.yml) and [documentation](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/shadowbox/server/api.yml) for more options.
 
-Remove an access key
-(e.g. remove access key 2)
-
-```
-curl --insecure -X DELETE $API_URL/access-keys/2
-```
+## Testing
 
-Set a data limit for all access keys
-(e.g. limit outbound data transfer access keys to 1MB over 30 days)
+### Manual
 
-```
-curl -v --insecure -X PUT -H "Content-Type: application/json" -d '{"limit": {"bytes": 1000}}' $API_URL/experimental/access-key-data-limit
-```
+1. Prerequisites
 
-Remove the access key data limit
+   - A locally built Docker image containing your modifications.
+   - Your Docker image uploaded to a registry (e.g., Docker Hub, Quay.io).
 
-```
-curl -v --insecure -X DELETE $API_URL/experimental/access-key-data-limit
-```
+1. Setup
 
-## Testing
+   - **Environment Variable:** Set `SB_IMAGE` to the uploaded image location. Example:
 
-### Manual
+     ```sh
+     export SB_IMAGE=yourdockerhubusername/shadowbox
+     ```
 
-After building a docker image with some local changes,
-upload it to your favorite registry
-(e.g. Docker Hub, quay.io, etc.).
+   - **Start the Server:**
 
-Then set your `SB_IMAGE` environment variable to point to the image you just
-uploaded (e.g. `export SB_IMAGE=yourdockerhubusername/shadowbox`) and
-run `npm run action server_manager/electron_app/start` and your droplet should be created with your
-modified image.
+     ```sh
+     npm run action shadowbox/docker/start
+     ```
 
 ### Automated
 
-To run the integration test:
-
-```
-npm run action shadowbox/integration_test/run
-```
+1. **Build and Run:**
 
-This will set up three containers and two networks:
+   ```sh
+   npm run action shadowbox/integration_test/run
+   ```
 
-```
-client <-> shadowbox <-> target
-```
+   This does the following:
 
-`client` can only access `target` via shadowbox. We create a user on `shadowbox` then connect using the Shadowsocks client.
+   - Sets up three containers (`client`, `shadowbox`, `target`) and two networks.
+   - Creates a user on `shadowbox`.
+   - Connects to `target` through `shadowbox` using a Shadowsocks `client`: `client <-> shadowbox <-> target`
 
-To test clients that rely on fetching a docker image from Dockerhub, you can push an image to your account and modify the
-client to use your image. To push your own image:
+1. **Testing Client Modifications:**
```

Comment:
This is not the client. It's the Manager. But we can just remove this section.

BTW, you can pass an image name to `npm run action shadowbox/integration_test/run`.

Reply:
Ah yeah my bad, I was thinking of "testing with clients" but I phrased it poorly. Let's just remove it .